mm2src/mm2_main/src/rpc/wc_commands/new_connection.rs

```diff
@@ -0,0 +1,32 @@
+use kdf_walletconnect::WalletConnectCtx;
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::prelude::*;
+use serde::{Deserialize, Serialize};
+
+use super::WalletConnectRpcError;
+
+#[derive(Debug, PartialEq, Serialize)]
+pub struct CreateConnectionResponse {
+    pub url: String,
+}
+
+#[derive(Deserialize)]
+pub struct NewConnectionRequest {
+    required_namespaces: serde_json::Value,
+    optional_namespaces: Option<serde_json::Value>,
```

Comment:
It would have been better to create concrete `Namespaces` type instead of just using `serde_json::Value`

Reply:
I see that we have `ProposeNamespaces` type that implements Serialize/Deserialize, why didn't you use it. Please do the same for similar requests and responses if possible.

Reply:
just didn't want to add the crate as a dep in `mm2_main` for just one/two imports